qa/tasks/cephfs/test_volumes.py

```diff
@@ -27,14 +27,20 @@ class TestVolumes(CephFSTestCase):
     def _fs_cmd(self, *args):
         return self.mgr_cluster.mon_manager.raw_cluster_cmd("fs", *args)
 
-    def _generate_random_subvolume_name(self):
-        return "{0}_{1}".format(TestVolumes.TEST_SUBVOLUME_PREFIX, random.randint(0, 10000))
+    def _generate_random_subvolume_name(self, count=1):
+        r = random.sample(range(10000), count)
+        subvolumes = ["{0}_{1}".format(TestVolumes.TEST_SUBVOLUME_PREFIX, c) for c in r]
+        return subvolumes[0] if count == 1 else subvolumes
```

Comment:
Just return `subvolumes` should work well. You are iterating the list in the caller anyway.